src/test/regress/expected/cte_inline_0.out

```diff
@@ -15,7 +15,7 @@ SHOW server_version \gset
 SELECT substring(:'server_version', '\d+')::int;
  substring
 ---------------------------------------------------------------------
-        11
+        12
```

Comment:
there is no (and shouldn't) have a difference in terms of cte_inlining between 12 and 13.

Can we get rid of one output file? We cann probably make `substring(:'server_version', '\d+')::int >= 12` for the version check

Reply:
I guess you are right, I remember adding this because there were some output changes but after some normalization etc there isn't any now:

```diff
talha@SANISANC002:~/citus/src/test/regress$ diff expected/cte_inline.out expected/cte_inline_0.out
18c18
<         13
---
>         12
775d774
<    9 | test9  | {"f1": 9, "f2": 162, "f3": "test9"}   |        1
776a776
>    9 | test9  | {"f1": 9, "f2": 162, "f3": "test9"}   |        1
```